scripts/tasks/backup

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Run a FogLAMP task written in Python
+if [ "${FOGLAMP_ROOT}" = "" ]; then
+	FOGLAMP_ROOT=/usr/local/foglamp
+fi
+
+if [ ! -d "${FOGLAMP_ROOT}" ]; then
+	logger "FogLAMP home directory missing or incorrectly set environment"
+	exit 1
+fi
+
+if [ ! -d "${FOGLAMP_ROOT}/python" ]; then
+	logger "FogLAMP home directory is missing the Python installation"
+	exit 1
+fi
+
+# We run the Python code from the python directory
+cd "${FOGLAMP_ROOT}/python"
```

Comment:
We do not need this, we set PYTHONPATH so will find the code there

Reply:
# Adds required paths for the execution of the python module
export PYTHONPATH="${FOGLAMP_ROOT}/python"